node/src/env0-deploy-flow.js

```diff
@@ -14,19 +18,22 @@ const assertRequiredOptions = (options) => {
 }
 
 const runDeployment = async (command, options, environmentVariables) => {
```

Comment:
Let's rename this maybe? It will run any command, not just deployment.

Reply:
5862a1d